deploy.sh

```diff
@@ -26,13 +31,24 @@ function iam_mapping {
     helm template aws ./helm --set aws.role.node="${node}",aws.role.user="${user}" | kubectl apply -n kube-system -f -
 }
 
+function get_kubeconfig {
+    eksctl utils write-kubeconfig hindsight-kubernetes-${ENVIRONMENT_NAME}
```

Comment:
Should this try to install eksctl or at least check to make sure the binary is installed and fail immediately if you don’t have it in your path?

Reply:
It will fail if it doesn't exist because this script does set -e.  That said, I'm going to leave this comment open to remind me to add a .toolversions file for eksctl as well.

Reply:
At a certain point its tough to imply what all tools exist as this script assumes you have aws, jq, openssl, eksctl, etc installed

Reply:
.tool-versions only really works if you have asdf managing your tools (and as a documentation hint).
does the `-e` bail out the script before any other actions have been taken? is it worth making the very first function definition in the script check for the required tools being in the path and then fail immediately with a helpful message before it has the potential to partially execute the script?

Reply:
Check my newest commit and see what you think.